app/styles/components/_horizontal-nav.scss

```diff
@@ -31,14 +31,14 @@
 
 
 .horizontal-nav__container {
-  overflow-x: auto;
+  padding-left: 25px;
   -ms-overflow-style: -ms-autohiding-scrollbar;
   -webkit-overflow-scrolling: touch;
   &::-webkit-scrollbar {
     display: none;
   }
 
-  background-color: rgba(0, 0, 0, 0);
+  //background-color: rgba(0, 0, 0, 0);
```

Comment:
delete commented code (unless you're planning to use it)

Reply:
Done.